src/main/java/eventprocessing/responseservices/ResponseService.java

```diff
@@ -0,0 +1,29 @@
+package eventprocessing.responseservices;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import eventprocessing.models.Message;
+import eventprocessing.models.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class ResponseService {
```

Comment:
`Response` and `ResponseService` sound very general, but it looks like they handle a particular kind of AWS API response. Have a think about the names to make it clearer.